docs/Branching_strategy.md

```diff
@@ -0,0 +1,68 @@
+## Branching strategy:
+
+This strategy is aimed at reducing the risk of production deploys. Smaller, more frequent deploys reduce the risk of each deploy. Since these smaller deploys are less likely to cause problems, that provides the benefit of less downtime and higher availability of your systems. It also has the benefit of better schedules for maintainers since there is rarely a need to wait until the site is not in use if you can do seamless deploys.
+
+So, the overall goal is to make sure that we have small, frequent deploys while ensuring proper quality checks. Before code is deployed it goes though 3 human evaluations and at least 3 rounds of automated testing.
```

Comment:
This paragraph uses "we" but the paragraph above uses "you", we might want to keep that consistent for clarity.